.github/workflows/linters-readme.yml

```diff
@@ -0,0 +1,54 @@
+# We need a separate workflow file so failures on the README are not reported on
+# all PRs, only on the ones that update the README.
+name: Linters (README.md)
+
+on:
+  # We need `pull_request_target` to access the secrets
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths: [README.md]
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  PYTHON_VERSION: '3.12'
+  NODE_VERSION: lts/*
+
+permissions:
+  contents: read
+
+jobs:
+  lint-readme-lists:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    if: ${{ github.repository_owner == 'nodejs' }}
```

Reply:
forks don't have the access needed to perform this workflow, right?

Reply:
Not really, the fork "just" needs to have a secret named `GH_USER_TOKEN` which would contain a token with sufficient permissions. I have this setup on my fork, so I could test this: https://github.com/aduh95/node/actions/runs/9981058233/job/27583833532?pr=9

Reply:
Other workflows restrict similar behavior, so I think this should do the same,

For example,
https://github.com/nodejs/node/blob/main/.github/workflows/comment-labeled.yml#L26C5-L26C43
https://github.com/nodejs/node/blob/main/.github/workflows/commit-queue.yml#L28

Reply:
No I think that's the wrong approach